### DISCUSSION TITLE
ae中做了个火箭曲线运动动画,设置了自动定向,但导出pag后自动定向不生效 火箭方向没沿着轨道

### DISCUSSION THREAD

**Discussion by CindyZhangxin** [Tencent/libpag#1850] [Q&A] [answered]
## 【版本信息】

4.2.63.71

## 【平台信息】

Android 原生 
## 【预期的表现】

![IMG_6096 HEIC](https://github.com/Tencent/libpag/assets/150134515/179b79ad-23f1-4b19-b408-3bca60553c69)

## 【实际的情况】

![IMG_6095 HEIC](https://github.com/Tencent/libpag/assets/150134515/452d01ab-f2e0-4a4d-b41b-7eec90c69a23)

**Comment by kevingpqi123** [comment, marked as answer]
如果是矢量的导出方式，需要关注可以直接导出的 AE 特性：https://pag.art/feature.html#ae
另外导出的过程中，不要放过导出过程中的提示和报错，按照导出插件给出到的提示来操作，就可以规避以及了解导出过程中的问题

这里又不需要图层编辑，可以使用 BMP 预合成的导出方式，这种方式支持所有 AE 特性
